auth0/resource_auth0_user.go

```diff
@@ -89,13 +89,13 @@ func newUser() *schema.Resource {
 			"user_metadata": {
 				Type:             schema.TypeString,
 				Optional:         true,
-				ValidateFunc:     validation.ValidateJsonString,
+				ValidateFunc:     validation.StringIsJSON,
 				DiffSuppressFunc: structure.SuppressJsonDiff,
 			},
 			"app_metadata": {
 				Type:             schema.TypeString,
 				Optional:         true,
-				ValidateFunc:     validation.ValidateJsonString,
+				ValidateFunc:     validation.StringIsJSON,
```

Comment:
validation.ValidateJsonString is deprecated: so we used StringIsJSON instead as it matches the exact signature. 

Docs:

```
// ValidateJsonString is a SchemaValidateFunc which tests to make sure the
// supplied string is valid JSON.
//
// Deprecated: use StringIsJSON instead
func ValidateJsonString(i interface{}, k string) (warnings []string, errors []error) {
	return StringIsJSON(i, k)
}
```